File: JAVASCRIPT/Cap06/c06ex04.js
// c06ex04.js

const rls = require('readline-sync');

let A = [];
let N, I;

N = rls.questionInt("Entre a quantidade de elementos da matriz: ");
console.log("");

for (I = 0; I <= N - 1; ++I) {
  A[I] = new Array(50).fill(' ').join('');

  A[I] = rls.question(`Entre o ${I + 1}o. nome: `);
}

console.log("");
console.log("Foram fornecidos os nomes:");
console.log("");

for (I = 0; I <= N - 1; ++I) {
  console.log(`Nome ${I + 1} = ${A[I]}`);
}

console.log("");
process.stdout.write("Tecle <Enter> para encerrar... ");
rls.question();

File: DART/Cap06/c06ex04.dart
// c06ex04.dart

import 'dart:io';

void main() {
  int N;
  List<String> A = [];

  stdout.write("Entre a quantidade de elementos da matriz: ");
  N = int.tryParse(stdin.readLineSync() ?? '') ?? 0;
  print("");

  for (int i = 0; i <= N - 1; i++) {
    stdout.write("Entre o ${i + 1}o. nome: ");
    A.add(stdin.readLineSync() ?? "");
  }

  print("");
  print("Foram fornecidos os nomes:");
  print("");

  for (int i = 0; i <= N - 1; i++) {
    print("Nome ${i + 1} = ${A[i]}");
  }

  stdout.write("\nTecle <Enter> para encerrar... ");
  stdin.readLineSync();
}
